components/Modules/module-detail-page.tsx

```
@@ -786,20 +806,25 @@ export default function ModuleDetailWrapper({
               </div>
             ) : (
               <p
-                className="text-muted-foreground cursor-pointer hover:bg-muted/50 p-4 rounded min-h-[158px]"
-                onClick={() => setIsEditingDescription(true)}
+                className="text-muted-foreground cursor-pointer hover:bg-muted/50 p-4 mr-4 mb-12 rounded min-h-[158px]"
+                onClick={() => setIsEditingContent(true)}
+                tabIndex={0}
+                onKeyDown={(e) => {
```

Biome (1.9.4)

[error] The HTML element p is non-interactive. Do not use tabIndex.

Adding non-interactive elements to the keyboard navigation flow can confuse users.
Unsafe fix: Remove the tabIndex attribute.


(lint/a11y/noNoninteractiveTabindex)